proposals/0157-mobile-choice-manager.md

```diff
@@ -95,9 +154,24 @@ The screen manager additions allow the developer to present a "dynamic" choice s
 
 ...
 
-- (void)presentDynamicChoiceSet:(SDLChoiceSetObject *)set mode:(SDLInteractionMode)mode;
-- (NSUInteger)uploadStaticChoiceSet:(SDLChoiceSetObject *)set;
-- (void)presentStaticChoiceSetWithId:(NSUInteger)id mode:(SDLInteractionMode)mode;
+// Return an error with userinfo [key: SDLChoiceCell, value: NSError] if choices failed to upload
+typedef void(^SDLPreloadChoiceCompletionHandler)(NSError *error);
+
+// The default keyboard configuration, this can be additionally customized per 
+@property (strong, nonatomic) SDLKeyboardProperties *keyboardConfiguration;
+
+// Key: cell.text, Value: cell
+@property (copy, nonatomic, readonly) NSDictionary<NSString *, SDLChoiceCell *> *preloadedChoices;
```

Comment:
I thought we agreed to a hash of the choice. `.text` is a very dangerous identifier.